src/config.js

```diff
@@ -14,7 +14,7 @@ try {
 
 global.config = {
   profiler: {
-    enabled: true
+    enabled: true,
```

Comment:
I like the `config_local.js` idea :+1: 

I think the default values should be the bot values. No profiler, no summary, no log, no nothing. So we can change that to false.